### PR TITLE
BUG fix to_pandas for nested objs

### PR DESCRIPTION
**Pull request by tgy**

### CODE REVIEW

**Comment by olivierteboul** [comment]
Thanks Valentin. However I would follow a different strategy.
First the function has little to do with pandas, since you only call the pd.DataFrame constructor at the end.
So what about having a function `def flatten(d: dict) -> dict` that flattens a dictionary. It can be defined in another module. Like `icubam.dict_utils.py` (does not exist yet)

Then for example `flatten({'k1': {'k2': 1, 'k3': 8}, 'k4': 6}) = {'k1_k2': 1, 'k1_k3': 8, 'k4': 6}`

Which mean that you could simply do:
```
def to_pandas(objs):
  return pd.DataFrame([dict_utils.flatten(obj.to_dict(max_depth=1)) for obj in objs])
```

what do you think?

**Comment by tgy** [comment]
@olivierteboul makes sense

**Comment by tgy** [comment]
@olivierteboul let me know

**Comment by rth** [comment]
FYI, there is a [`json_normalize`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.json_normalize.html) in pandas that does this:
```py
>>> from pandas.io.json import json_normalize

>>> data = [{'id': 1, 'name': {'first': 'Coleen', 'last': 'Volk'}},
>>>        {'name': {'given': 'Mose', 'family': 'Regner'}},
>>>        {'id': 2, 'name': 'Faye Raker'}]

>>> json_normalize(data)
    id        name name.family name.first name.given name.last
0  1.0         NaN         NaN     Coleen        NaN      Volk
1  NaN         NaN      Regner        NaN       Mose       NaN
2  2.0  Faye Raker         NaN        NaN        NaN       NaN
```

(I also happened to re-implement something similar for another project before discovering this function).

**Comment by tgy** [comment]
@rth you're a rock star. thanks.

File: icubam/www/handlers/db.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from absl import logging  # noqa: F401
+import pandas as pd
```
Comment (rth) [inline icubam/www/handlers/db.py:2]: Unused import

File: icubam/db/store.py [hunk 2]
```diff
@@ -736,5 +736,7 @@ def create_store_for_sqlite_db(cfg) -> Store:
   return Store(engine, salt=cfg.DB_SALT)
 
 
-def to_pandas(objs) -> pd.DataFrame:
-  return pd.DataFrame([x.to_dict() for x in objs])
+def to_pandas(objs):
+  return pd.io.json.json_normalize(
```
Comment (olivierteboul) [inline icubam/db/store.py:740]: good job: from a complex function to a one-liner!! awesome

**Comment by olivierteboul** [comment]
:clap: you could add test... but you can also merge.

**Comment by rth** [comment]
Let's merge then. Just need to rebase @tgy . Thanks for the fix!

**Comment by tgy** [comment]
i tested it locally and it works when pulling the data. should be ok

**Pull request merged**